modules/administration-guide/pages/removing-user-data-in-compliance-with-the-gdpr.adoc

```diff
@@ -7,34 +7,29 @@
 [id="removing-user-data-in-compliance-with-the-gdpr"]
 = Removing user data in compliance with the GDPR
 
-You can remove a user’s data in compliance with the link:https://gdpr.eu/[General Data Protection Regulation (GDPR)] that enforces the right of individuals to have their personal data erased.
+You can remove a user’s data on {ocp} in compliance with the link:https://gdpr.eu/[General Data Protection Regulation (GDPR)] that enforces the right of individuals to have their personal data erased. The process for other {kubernetes} infrastructures might vary. Follow the user management best practices of the provider you are using for the {prod} installation.
 
 WARNING: Removing user data as follows is irreversible! All removed data is deleted and unrecoverable!
 
 .Prerequisites
 
-* An active session with administrative permissions for {prod-short}. See xref:configuring-the-administrative-user.adoc[].
-
-* An active `oc` session with administrative permissions for the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+* An active `oc` session with administrative permissions for the {ocp} cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
```

Comment:
This uses `{ocp4-ver}` parameter. @deerskindoll how would you point to the latest without hard-coding version in the URL ?

Reply:
you use `latest` instead of `{ocp4-ver}` in the link:

https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html

Reply:
good catch, let's use latest :+1:

Reply:
sure!